ipykernel/kernelspec.py

```diff
@@ -61,35 +61,45 @@ def get_kernel_dict(extra_arguments=None):
 
 def write_kernel_spec(path=None, overrides=None, extra_arguments=None):
     """Write a kernel spec directory to `path`
-    
+
     If `path` is not specified, a temporary directory is created.
     If `overrides` is given, the kernelspec JSON is updated before writing.
-    
+    if 'resources' is given, copies resources from non-standard location.
+
```

Comment:
this should be removed, since resources were added, but then removed as a parameter to write_kernel_spec in this PR.